content/zh/docs/languages/go/getting-started.md

```diff
@@ -0,0 +1,1145 @@
+---
+title: 开始
+weight: 10
+default_lang_commit: 3512b0ae11f72d3a954d86da59ad7f98d064bdad
+# prettier-ignore
+cSpell:ignore: chan fatalln funcs intn itoa khtml otelhttp rolldice stdouttrace strconv
+---
+
+<!-- markdownlint-disable blanks-around-fences -->
+<?code-excerpt path-base="examples/go/dice"?>
+
+本页面将向你展示如何在 Go 中开始使用 OpenTelemetry。
+
+你将学习如何手动为一个简单的应用程序进行插桩, 并且使其能将 [traces][]，[metrics][] 和 [logs][] 输出到终端。
+
+{{% alert title="注意" %}}
+
+日志信号仍处于实验阶段，在未来版本可能会引入不兼容的更改。
+
+{{% /alert %}}
+
+## 前置条件{#prerequisites}
+
+确保你本地已经安装了：
+
+- [Go](https://go.dev/) 1.22 及更高版本。
+
+## 示例应用程序{#example-application}
+
+以下示例使用了一个基本的 [`net/http`](https://pkg.go.dev/net/http) 应用程序。
+如果你没有使用 `net/http` 也没关系 —— 你仍然可以在其他 Web 框架中使用
+OpenTelemetry Go，比如 Gin 和 Echo，有关支持框架的完整库列表，请参阅
+[registry](/ecosystem/registry/?component=instrumentation&language=go)。
+
+如果需要更复杂的示例，请参阅 [examples](/docs/languages/go/examples/)。
+
+### 开始{#setup}
+
+首先，需要在新的目录中设置 `go.mod`：
+
+```shell
+go mod init dice
+```
+
+### 创建并启动 HTTP 服务器{#create-and-launch-an-http-server}
+
+在同一个文件夹中，创建一个 `main.go` 文件，并添加以下代码到这个文件中：
+
+```go
+package main
+
+import (
+    "log"
+    "net/http"
+)
+
+func main() {
+    http.HandleFunc("/rolldice", rolldice)
+
+    log.Fatal(http.ListenAndServe(":8080", nil))
+}
+```
+
+创建另一个叫做 `rolldice.go` 的文件，并添加以下代码到该文件。
+
+```go
+package main
+
+import (
+    "io"
+    "log"
+    "math/rand"
+    "net/http"
+    "strconv"
+)
+
+func rolldice(w http.ResponseWriter, r *http.Request) {
+    roll := 1 + rand.Intn(6)
+
+    resp := strconv.Itoa(roll) + "\n"
+    if _, err := io.WriteString(w, resp); err != nil {
+        log.Printf("Write failed: %v\n", err)
+    }
+}
+```
+
+通过以下命令构建并运行应用程序。
+
+```shell
+go run .
+```
+
+在你的浏览器中打开 <http://localhost:8080/rolldice> 确保应用程序正常工作。
+
+## 添加 OpenTelemetry 插桩{#add-opentelemetry-instrumentation}
+
+现在我们将展示如何向示例应用程序添加 OpenTelemetry 插桩。如果你正在使用自己的应用程序，也可以按照步骤操作，只需要注意你的代码可能会略有不同。
+
+### 添加依赖项{#add-dependencies}
+
+安装以下软件包：
+
+```shell
+go get "go.opentelemetry.io/otel" \
+  "go.opentelemetry.io/otel/exporters/stdout/stdoutmetric" \
+  "go.opentelemetry.io/otel/exporters/stdout/stdouttrace" \
+  "go.opentelemetry.io/otel/exporters/stdout/stdoutlog" \
+  "go.opentelemetry.io/otel/sdk/log" \
+  "go.opentelemetry.io/otel/log/global" \
+  "go.opentelemetry.io/otel/propagation" \
+  "go.opentelemetry.io/otel/sdk/metric" \
+  "go.opentelemetry.io/otel/sdk/resource" \
+  "go.opentelemetry.io/otel/sdk/trace" \
+  "go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"\
+  "go.opentelemetry.io/contrib/bridges/otelslog"
+```
+
+这将安装 OpenTelemetry SDK 组件以及 `net/http` 的插桩库。
+
+如果你要为其他网络请求库进行插桩，需要安装对应的插桩库，更多信息请查看
+[libraries](/docs/languages/go/libraries/)。
+
+### 初始化 OpenTelemetry SDK{#initialize-the-opentelemetry-sdk}
+
+首先我们需要初始化 OpenTelemetry SDK。这对任何需要导出遥测数据的应用程序都是 _必须_ 的
```

Comment:
[Italic doesn't work well with Chinese](https://en.zhihu.com/answer/14040639), I think we should avoid it.

```suggestion
首先我们需要初始化 OpenTelemetry SDK。这对任何需要导出遥测数据的应用程序都是**必须**的
```